plugins/zfs/zfs_count

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Could you think of a more precise name for this plugin? (i.e. what exactly is *counted*)

Reply:
renamed